nixos/lib/make-options-doc/default.nix

```diff
@@ -91,9 +91,8 @@
   lib,
   options,
   transformOptions ? lib.id, # function for additional transformations of the options
-  documentType ? "appendix",
-  # TODO deprecate "appendix" in favor of "none"
-  #      and/or rename function to moduleOptionDoc for clean slate
+  documentType ? null,
+  # TODO  rename function to moduleOptionDoc for clean slate
```

Comment:
I don't think this is relevant anymore. IIUC it was included as an _alternative_ to deprecating the argument. If we're deprecating and eventually removing it, then this TODO is no longer relevant.